immutables/_map.c

```diff
@@ -3363,6 +3363,11 @@ map_reduce(MapObject *self)
     return tup;
 }
 
+static PyObject *
+map_py_class_getitem(PyObject *type, PyObject *item)
+{
+    return type;
```

Comment:
There's a ref leak here, add Py_INCREF

Reply:
Or better yet just use Py_RETURN_NONE macro

Reply:
I tried following [advice I got from](https://gitter.im/python/typing?at=5d25bd59cdb70805c96e239f) Ivan in the mypy team:

> (also it should return class object itself, not None)

Would this be the correct usage of `Py_INCREF`?

Reply:
Yes.

Reply:
Also are you fine with that implementation as opposed to returning None? :)

Reply:
Yes! :)